src/domain/chains/chains.validator.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { HttpException } from '@nestjs/common';
+import { JsonSchemaService } from '../schema/json-schema.service';
+import { ValidationErrorFactory } from '../schema/validation-error-factory';
+import { ChainsValidator } from './chains.validator';
+import chainFactory from './entities/__tests__/chain.factory';
+
+const mockValidationErrorFactory = jest.mocked({
+  from: jest.fn(),
+} as unknown as ValidationErrorFactory);
+
+const validationFunction = jest.fn();
+const mockJsonSchemaService = jest.mocked({
+  addSchema: jest.fn(),
+  compile: jest.fn().mockImplementation(() => validationFunction),
+} as unknown as JsonSchemaService);
+
+describe('Chains validator', () => {
+  const validator = new ChainsValidator(
+    mockValidationErrorFactory,
+    mockJsonSchemaService,
+  );
+
+  const chain = chainFactory();
+  const chains = [chainFactory(), chainFactory()];
```

Comment:
The same here we should move this to the test

Reply:
Done in [ae890e7b0a9f73e8ea70d1ed2b7bf12f2109481a](https://github.com/5afe/safe-client-gateway-nest/commit/ae890e7b0a9f73e8ea70d1ed2b7bf12f2109481a)